src/Components/App.js

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import '../App.css';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
 import AdminPage from './Admin/AdminPage';
+import drawer from './Drawer/drawer';
 
 function App() {
   return (
     <div className="App">
       <BrowserRouter>
         <Switch>
-          <Route path ="/admin" exact component={AdminPage}/>
+          <Route path ="/admin" exact component={AdminPage}/> 
+          <Route path="/drawer" component={drawer} />
```

Comment:
I imagine the drawer component would belong outside of the Switch statement, as it's a common feature for all pages. The drawer component probably should not have it's own page.
I would recommend just inserting the component above line 11.
```javascript
<BrowserRouter>
  <drawer>
  <Switch>
    ...
  </Switch>
</BrowserRouter>
```

Reply:
Also, we should follow the naming convention of having component names that are capitalized. So "Drawer.js" instead of "drawer.js", and `function Drawer() {}` instead of function `drawer() {}`